src/main/java/jupyter/AsDisplayData.java

```diff
@@ -0,0 +1,15 @@
+package jupyter;
+
+import java.util.Map;
+
+/**
+ * Convenience interface for classes whose display data is known in advance.
+ *
```

Comment:
Javadoc nit: if you intend to have a paragraph break, you have to add `<p>`

Reply:
Fixed.